infra/terraform/modules/lambda_mgmt/variables.tf

```diff
@@ -0,0 +1,45 @@
+
+variable "lambda_function_name" {
+  description = "The default name of all resources"
+}
+
+variable "lambda_runtime" {
+  default     = "go1.x"
+  description = "Runtime language for lambda function. See: https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime"
+}
+
+variable "handler" {
+  description = "Entrypoint filename. For `Golang` this must match `lambda_function_name`"
+}
+
+variable "zipfile" {
+  description = "Path to zip file containing code"
+}
+
+variable "enabled" {
+  default = true
+}
+
+variable "timeout" {
+  default = 3
+}
+
+variable "schedule_expression" {
+  default = "rate(1 day)"
```

Comment:
I suppose we don't care too much when the snapshots/invocation happen.

Reply:
At first, I didn't but I supposed 10 or 11 PM could be sensible. I'll leave this as a default and change at module invocation